roles/calico_master/tasks/main.yml

```diff
@@ -15,6 +15,12 @@
     resource_name: privileged
     state: present
 
+- name: Set default selector for kube-system
```

Comment:
That doesn't look right. 

We're creating static pods in `kube-system`, the namespace certainly has empty selector. So either it get changed later on - and needs to be fixed there - or something else causes this.

Do you have ansible logs for the failed installation?